conda-recipe/meta.yaml

```diff
@@ -18,7 +18,8 @@ requirements:
       - wheel
     build:
       - {{ compiler('cxx') }}
-      - {{ compiler('dpcpp') }}  >=2023.0  # [not osx]
+      - {{ compiler('dpcpp') }}  =2023.1.0  # [not osx]
+      - sysroot_linux-64 >=2.17  # [linux]
```

Comment:
Is this problem still not fixed in the new compiler build? Is it possible to test this?

Reply:
If we are pinning compiler to 2023.1, we have to pin `sysroot_linux-64` also, otherwise `2.12` will be used for building.
The PR is about to temporary hide test crashes with messages like:
```
Abort was called at 192 line in file:
/opt/src/l0_gpu_driver/shared/source/kernel/kernel_arg_descriptor.h
```
when the latest 2023.2 compiler is used, until the issue is resolve, and so unblock the internal CI.
The idea is to build with 2023.1 but run with 2023.2 runtime.